nightfall-optimist/src/services/block-assembler.mjs

```diff
@@ -79,7 +80,9 @@ export async function conditionalMakeBlock(proposer) {
     or we're no-longer the proposer (boo).
    */
 
-  logger.info(`I am the current proposer: ${proposer.isMe}`);
+  if (proposer.isMe) {
```

Comment:
The next `if` statament is equal to this one, please, move the log statement to there and remove this `if`.

Reply:
done